bool ready
int8 ID

---

bool ready
geometry_msgs/PoseStamped new_point